### PR TITLE
Add main footer navigation

### PR DESCRIPTION
**Pull request by Renny4Real**
- Add main footer navigation as described in the [spreadsheet](https://docs.google.com/spreadsheets/d/141spueIa6sG7Eyv6EjsqFn8s4ubJ8YX0LzpS0kGT91Q/edit#gid=696358261).

- Add styling as necessary

BEFORE:

Screenshot 2019-04-29 at 10 48 04

------------------------------------------------------------------------------------------------------

AFTER:
Screenshot 2019-04-29 at 10 43 35

------------------------------------------------------------------------------------------------------

**Any feedback/suggestions would be appreciated.**

### CODE REVIEW

**Comment by georgeschena** [comment]
It could be my eyes playing up, but it looks like the spacing between the `About` and `Connect` column are not the same as the gap between the `Connect` and the `Admin` column. But I could be wrong

**Comment by Renny4Real** [comment]
**UPDATE:**
- Adjust footer column spacing. 

**NEW DISPLAY:**
Screenshot 2019-04-29 at 11 40 34

**Any feedback/suggestions would be appreciated.**

------------------------------------------------------------------------------------------------------
> Note: Out of the three `<div> classes` corresponding to each column section, only two were changed.
> 
> Not all three were changed because changing all of them resulted in the break line beneath the columns to rising too close to the footer columns. **_SEE BELOW:_**
> Screenshot 2019-04-29 at 11 43 58
>

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -65,10 +65,45 @@
...
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Organisations using GovWifi', '/about-govwifi/organisations-using-govwifi' %>
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Support', '/support' %>
+              </li>
+            </ul>
+          </div>
+          <div class="govuk-footer__section govuk-footer__list--columns-1">
```
Comment (tommotaylor) [inline source/layouts/layout.erb]: @Renny4Real this should also be a `column-one-third` if you are using that class on the other columns. Or does that not work?

Comment (tommotaylor) [inline source/layouts/layout.erb]: This feels a bit hacky.

You can use the margin class on the wrapping div (L67) to get the desired bottom margin:
```
        <div class="govuk-footer__navigation govuk-!-margin-bottom-6">
```

Comment (tommotaylor) [reply source/layouts/layout.erb]: We should be using 3x `column-one-thirds`

**Comment by Renny4Real** [comment]
**UPDATE:**

- Change all three column div to have the same classes.
- Add bottom margin to wrapping div surrounding the columns.

**NEW DISPLAY:**
Screenshot 2019-04-29 at 12 18 20

**Pull request merged**